apertium-tur-uzb.tur-uzb.dix

```diff
@@ -3660,6 +3686,258 @@
+<e>       <p><l>Amerikan<b/>Futbol<b/>Ligi<s n="np"/><s n="org"/></l>                  <r>Amerika<b/>futbol<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Arap<b/>Ligi<s n="np"/><s n="org"/></l>                                <r>Arab<b/>Ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Arjantin<b/>Futbol<b/>Ligi<s n="np"/><s n="org"/></l>                  <r>Argentina<b/>futbol<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Arjantin<b/>Ligi<s n="np"/><s n="org"/></l>                            <r>Argentina<b/>Ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Arnavutluk<b/>Süper<b/>Ligi<s n="np"/><s n="org"/></l>                 <r>Albaniya<b/>Superligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Asya<b/>Şampiyonlar<b/>Ligi<s n="np"/><s n="org"/></l>                 <r>Osiyo<b/>chempionlar<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Atletizm<b/>Ligi<s n="np"/><s n="org"/></l>                            <r>Atletika<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avrupa<b/>Komisyonu<s n="np"/><s n="org"/></l>                         <r>Yevropa<b/>komissiyasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avrupa<b/>Ligi<s n="np"/><s n="org"/></l>                              <r>Yevropa<b/>Ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avrupa<b/>Parlamentosu<s n="np"/><s n="org"/></l>                      <r>Yevropa<b/>parlamenti<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avrupa<b/>Şampiyonlar<b/>Ligi<s n="np"/><s n="org"/></l>               <r>Yevropa<b/>chempionlar<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avrupa<b/>ve<b/>Dış<b/>Politika<b/>Vakfı<s n="np"/><s n="org"/></l>    <r>Yevropa<b/>va<b/>tashqi<b/>siyosat<b/>jamgʻarmasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avustralya<b/>Futbol<b/>Ligi<s n="np"/><s n="org"/></l>                <r>Avstraliya<b/>futbol<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Avusturya<b/>Basketbol<b/>Ligi<s n="np"/><s n="org"/></l>              <r>Avstriya<b/>basketbol<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Ayrıca<b/>Kıbrıs<b/>Dördüncü<b/>Ligi<s n="np"/><s n="org"/></l>        <r>Kipr<b/>Toʻrtinchi<b/>Ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Ayrıca<b/>Kıbrıs<b/>Üçüncü<b/>Ligi<s n="np"/><s n="org"/></l>          <r>Kipr<b/>Uchinchi<b/>Ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Ayrıca<b/>Yugoslavya<b/>Ligi<s n="np"/><s n="org"/></l>                <r>Yugoslaviya<b/>ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Azerbaycan<b/>Premier<b/>Ligi<s n="np"/><s n="org"/></l>               <r>Ozarbayjon<b/>Premer%-ligasi<s n="np"/><s n="org"/></r></p></e>
+<e>       <p><l>Başta<b/>İngiltere<b/>Premier<b/>Ligi<s n="np"/><s n="org"/></l>       <r>Angliya<b/>Premer%-ligasi<s n="np"/><s n="org"/></r></p></e>
```

Comment:
You don't need to put başta here jut add it as İngiltere<b/>Premier<b/>Ligi

Reply:
Fixed it. The reason I added it in the first place is because this entry exists in tur.lexc and my script automatically tries to find entries(of a given type) and creates bidix format(if not already exists in bidix), so I manually translate them and add to bidix(Google Translate API also helps me translate fast as a sidekick).  Yet another script takes that bidix and makes a list of uzbek lexc entries to add to uzb.lexc(if not already exists).